internal/pkg/client/unstructured/helpers.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unstructured
+
+import (
+	"fmt"
+	api_unstructured "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"strings"
+)
+
+func jsonPath(fields []string) string {
+	return "." + strings.Join(fields, ".")
+}
+
+// NestedInt returns the int value of a nested field.
+// Returns false if value is not found and an error if not an int
+func NestedInt(obj map[string]interface{}, fields ...string) (int, bool, error) {
```

Comment:
[lodash](https://lodash.com/docs/4.17.11#get) is a good example of these types of functions.

How about calling this `GetInt`

Reply:
havent added jsonpath yet. But did something similar which can be swapped with json implementation. We get an unstructured which will not work for jsonpath.